climate_categories/data/conversion.FAO.IPCC2006_PRIMAP.csv

```diff
@@ -0,0 +1,20 @@
+# references: FAOSTAT https://files-faostat.fao.org/production/GT/Mapping_to_UNFCCC_IPCC.pdf
```

Comment:
General comment: I think we should try working with the mapping to see where we run into problems with different data coverage and refine the mapping based on the tests.

I can't judge if the mapping is complete just from looking at the file here, but you tested that by comparing results, correct?